pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -96,7 +96,7 @@ func Apply(clientSet clientset.Interface, target *deployments.Target) error {
 			if err := node.AddTargetInformationToInitConfigurationWithClusterVersion(target, initCfg, nodeVersionInfoUpdate.Update.APIServerVersion); err != nil {
 				return errors.Wrap(err, "error adding target information to init configuration")
 			}
-			kubeadm.SetContainerImagesWithClusterVersion(initCfg, nodeVersionInfoUpdate.Update.APIServerVersion)
+			kubeadm.UpdateConfigurationWithClusterVersion(initCfg, nodeVersionInfoUpdate.Update.APIServerVersion)
```

Comment:
If you decide to keep the `UpdateConfigurationWithClusterVersion` function to return `error` then it makes sense to check for it as well. I would be in favor of dropping all of your `error` returns in the following functions:

- `UpdateConfigurationWithClusterVersion`
- `setContainerImagesWithClusterVersion`
- `setContainerImagesWithClusterVersion`

this is because I think they will always return `nil`.

Reply:
I think you reviewed an older version here, I updated that already :), not your fault, I've been force pushing a lot lately on this one!